edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/RouteTest.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.Devices.Routing.Core.Test
     using Microsoft.Azure.Devices.Routing.Core.MessageSources;
     using Microsoft.Azure.Devices.Routing.Core.Query;
     using Microsoft.Azure.Devices.Routing.Core.Test.Endpoints;
+    using Moq;
     using Xunit;
 
     [ExcludeFromCodeCoverage]
```

Comment:
Since this file is changed, I think [ExcludeFromCodeCoverage] should be removed.

Reply:
doesn't this mean the testcode shouldn't be included in CodeCoverage?  If so, seems like that's still what we want.

---
In reply to: [377461886](https://github.com/Azure/iotedge/pull/2469#discussion_r377461886) [](ancestors = 377461886)

Reply:
talked and resolved offline

---
In reply to: [377894483](https://github.com/Azure/iotedge/pull/2469#discussion_r377894483) [](ancestors = 377894483,377461886)